index.html

```diff
@@ -132,120 +170,135 @@ <h2 class="card-title">Tonic</h2>
+                <div class="skills-arrow">
+                  <h3>Skills</h3>
+                  <img src="./images/arrow2.png" alt="Arrow right">
+                </div>
+                <div class="line"></div>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
     </section>
 
     <footer class="con-form-wrapper">
-      <div class="contact-form">
-        <h2 class="contact-me">Contact me</h2>
-        <p class="contact-description">If you have an application you are interested in developing, a feature that you
-          need built or a project that needs coding. I’d love to help with it.</p>
-        <form id="fs-frm" name="simple-contact-form" accept-charset="utf-8" action="https://formspree.io/f/mrgjgzbv"
-          method="post">
-          <div class="frm-inputs-cont">
-            <label for="full-name"></label>
-            <input class="con-frm-input" type="text" name="name" id="full-name" placeholder="First and Last Name"
-              required maxlength="30">
-            <label for="email-address"></label>
-            <input class="con-frm-input" type="email" name="_replyto" id="email-address"
-              placeholder="email@microverse.org" required>
-            <label for="message"></label>
-            <textarea class="con-frm-mess" rows="5" name="message" id="message" placeholder="Write your message here"
-              required maxlength="500"></textarea>
-            <input type="hidden" name="_subject" id="email-subject" value="Contact Form Submission">
-            <input class="frm-button" type="submit" value="Get In Touch">
-          </div>
-        </form>
+      <div class="footer-connect-container">
+        <div class="contact-form">
+          <h2 class="contact-me">Contact me</h2>
+          <p class="contact-description">If you have an application you are interested in developing, a feature that you
+            need built or a project that needs coding. I’d love to help with it.</p>
+          <form id="fs-frm" name="simple-contact-form" accept-charset="utf-8" action="https://formspree.io/f/mrgjgzbv"
+            method="post">
+            <div class="frm-inputs-cont">
+              <label for="full-name"></label>
+              <input class="con-frm-input" type="text" name="name" id="full-name" placeholder="First and Last Name"
+                required maxlength="30">
+              <label for="email-address"></label>
+              <input class="con-frm-input" type="email" name="_replyto" id="email-address"
+                placeholder="email@microverse.org" required>
+              <label for="message"></label>
+              <textarea class="con-frm-mess" rows="5" name="message" id="message" placeholder="Write your message here"
+                required maxlength="500"></textarea>
+              <input type="hidden" name="_subject" id="email-subject" value="Contact Form Submission">
+              <input class="frm-button" type="submit" value="Get In Touch">
```

Comment:
## [OPTIONAL]

- [ ] Nice following button interactions but I think the design would look much better if your change the cursor to the pointer when hovering the button `get in touch ` in the form
- [ ] The page title is adequately and briefly describes the content of the page. For example:

- Poor titles:
 > Microverse project | Your portfolio.
 > Portfolio project
- Better page titles:
 > Mark's portfolio
 > Mark full-stack web developer